Source/WebCore/css/CSSPendingSubstitutionValue.cpp

```diff
@@ -33,7 +33,7 @@ namespace WebCore {
 
 RefPtr<CSSValue> CSSPendingSubstitutionValue::resolveValue(Style::Builder& builder, CSSPropertyID propertyID) const
 {
-    auto cacheValue = [&](auto data) {
+    auto cacheValue = [this, protectedThis = Ref { *this }](auto data) {
```

Comment:
The lambda parameter to `resolveAndCacheValue()` is marked as `NOESCAPE` so this shouldn't be needed. If static analysis is complaining, I suggest we get rid of this local variable and pass in the lambda directly to `resolveAndCacheValue()`. Then I'm pretty sure the static analysis would be happy with capturing just `this`.